functions/snippets/pom.xml

```diff
@@ -8,6 +8,18 @@
   <artifactId>functions-snippets</artifactId>
   <version>1.0-SNAPSHOT</version>
 
+  <packaging>pom</packaging>
```

Comment:
This may execute multiple times in J-D-S as it does a search for pom.xml files, and not this style.

(ie. this might be redundant)

Reply:
This file was deleted, per @kurtisvg's suggestion below. 🙂